### ISSUE TITLE
Get Balance from API

### ISSUE THREAD

**Issue by gedanziger** [bitcoingreen/bitg-webwallet#11]
After login, have a service call get the balance of the initialized address from this api call:

```
https://www.coinexplorer.net/api/v1/BITG/address/balance?address=Gabc
```

Until bitg-js lib is implemented, use this param for `address` above: `GTsqojGaG2sy4uUTwyqwjxDtaVaF9ja5DV`. It should be pulled from the application state set in #10 .

**Comment by gedanziger** [comment]
I checked the call made in the console tab. @kdbusiness121 @oleg-baturov I get this response obj:

```
{
    "success": true,
    "result": {
        "GTsqojGaG2sy4uUTwyqwjxDtaVaF9ja5DV": "20.00000000"
    },
    "error": null
}
```

but balance isn't update to the screen.

**Comment by 0xweb3builder** [comment]
fixed by e382312142644183e27fbb5d413a2af8184a068e
